app/src/main/res/values-ca/strings.xml

```diff
@@ -1365,19 +1365,19 @@
     <string name="privacy_policy"><![CDATA[<b>Privadesa:</b><br/>Suntimes no coŀlecta, emmagatzema, o transmet dades personals. <b>No contè anuncis</b>, <b>càlculs estadístics</b>, <b>rastrejadors</b>, i <b>no utilitza permissos innecessaris</b>. <br/><br/>L\'aplicació es configura amb dades de localització. Aquesta informació (i qualsevol altra dada configurada emprada per l\'aplicació) s\'emmagatzema localment i no surt del dispositiu.<br/><br/><xliff:g id="permissionsExplained">%1$s</xliff:g>]]></string>
     <string name="privacy_permission_location"><![CDATA[Suntimes utilitza el permís de <b>localització</b> per determinar la teva longitud i latitud.<br/><br/>Aquest permís és opcional; la localització es configura manualment per defecte.]]></string>
     <string name="privacy_permission_storage"><![CDATA[Suntimes utilitza el permís d\'<b>emmagatzematge extern</b> per fer una còpia de seguretat o exportar dades a un arxiu (p. ex. llocs, temes).]]></string>
-    <string name="privacy_permission_storage1"><![CDATA[Alarmes Suntimes utilitza el permís d\'<b>emmagatzematge extern</b> per reproduïr sons localitzats en la targeta SD.<br/><br/> Aquest permís és opcional; es reproduirà el so predeterminat si l\'accès és denegat. Aquest permís no és un requisit per als sons del sistema (emmagatzematge multimèdia).]]></string>    <!-- TODO -->
+    <string name="privacy_permission_storage1"><![CDATA[Alarmes Suntimes utilitza el permís d\'<b>emmagatzematge extern</b> per reproduïr sons localitzats en la targeta SD.<br/><br/> Aquest permís és opcional; es reproduirà el so predeterminat si l\'accès és denegat. Aquest permís no és un requisit per als sons del sistema (emmagatzematge multimèdia).]]></string>    
     <string name="privacy_permissiondialog_prompt"><![CDATA[Demanar permisos ara?]]></string>
     <string name="privacy_permissiondialog_ignore">Ignorar</string>
     <string name="privacy_permissiondialog_title">Permisos</string>
-    <string name="privacyTab">Privacy</string>   <!-- TODO -->
+    <string name="privacyTab">Privacitat</string>   
 
     <string name="security_dialog_title">Alerta de seguretat</string>  
     <string name="security_duplicate_permissions">Un permís que pertany a Suntimes:\n\n<xliff:g id="permissionName" example="suntimes.permission.READ_CALENDAR">%1$s</xliff:g>\n\ntambé està definit per: <xliff:g id="packageName" example="some.other.app">%2$s</xliff:g></string>  
     <string name="feature_not_supported_by_api">Aquesta caracteristica no està suportada per l\'API %1$s.</string>
     <string name="feature_not_supported_by_source">Aquesta caracteristica no està suportada per la font de dades actual.</string>
     <string name="reboot_required_message">És possible que es necessiti reiniciar per fer efectius els canvis.</string>
 
-    <string name="dialog_ok">OK</string>  <!-- TODO -->
+    <string name="dialog_ok">D\'acord</string>  
     <string name="dialog_cancel">Canceŀlar</string>
```

Comment:
Should this value also be changed to `Canceŀla`?

Reply:
"Canceŀlar" is the verb "cancel". "Canceŀla" is the second person of present simple of that verb.

"D'acord" means "Agree". It could be "Acceptar", too.